pkg/appmanager/appManager.go

```diff
@@ -96,20 +96,22 @@ type Manager struct {
 	eventRecorder record.EventRecorder
 	eventSource   v1.EventSource
 	// Route configurations
-	routeConfig RouteConfig
+	routeConfig       RouteConfig
+	nodeLabelSelector string
```

Comment:
I would add a newline or comment between these two fields (just per the convention of the struct).

Reply:
+1